crates/sound/src/device.rs

```diff
@@ -397,6 +397,10 @@ impl VhostUserSoundThread {
 
         let mut stream_ids = BTreeSet::default();
 
+        // temporary workaround for processing the buffers speed
```

Comment:
This should be set in a separate commit for historical purposes & it can then be reverted cleanly once the issue is resolved

Reply:
noted